charts/tidb-lightning/templates/job.yaml

```diff
@@ -125,8 +128,10 @@ spec:
       {{ else }}
       - name: credentials
         secret:
+          {{- if .Values.dataSource.remote.secretName}}
```

Comment:
If secretName not configured, then this volume is not required, neither the volumeMount.

Reply:
done